substratevm/src/com.oracle.svm.core.graal.llvm/src/com/oracle/svm/core/graal/llvm/LLVMNativeImageCodeCache.java

```diff
@@ -438,17 +479,75 @@ private void llvmOptimize(DebugContext debug, String outputPath, String inputPat
 
     private void llvmCompile(DebugContext debug, String outputPath, String inputPath) {
         try {
+            LLVMUtils.Target arch;
+            if (targetPlatform instanceof Platform.AMD64) {
+                arch = LLVMUtils.Target.AMD64;
+            } else if (targetPlatform instanceof Platform.AArch64) {
+                arch = LLVMUtils.Target.AArch64;
+            } else {
+                throw unsupportedFeature("Unknown target");
```

Comment:
Errors reporting "unknown value" errors should always include the unknown value.